src/freertos/STM32F769I-DISCO/Src/syscalls.c

```diff
@@ -1,3 +1,18 @@
+/*********************************************************************
```

Comment:
This file was generated by STM32Cube, best to leave it as it was I think.

Reply:
Done